### DISCUSSION TITLE
Default a Note to open in "Note Map" view?

### DISCUSSION THREAD

**Discussion by devlinjunker** [TriliumNext/Trilium#3522] [Q&A] [answered]
Is it possible to set a note to open in "Note Map" View? Maybe some tag that I am not aware of? (#defaultView=NoteMap) or something?

i.e. always default to this view when opening the `Knowledge Map` note?
<img width="889" alt="Screen Shot 2023-01-13 at 11 48 23 AM" src="https://user-images.githubusercontent.com/1504590/212406241-cae7f93a-4f80-4baf-9906-e486440c3fed.png">

**Comment by zadam** [comment, marked as answer]
That's not currently possible, but another way to achieve the same is to create a note of type "Note Map":

![image](https://user-images.githubusercontent.com/617641/212409865-a39e6661-08e5-44f9-af8c-25d0e42dc544.png)

By default it displays visualization of the parent note, but can be configured with `#mapRootNoteId` label to either "hoisted" (visualises current hoisted subtree) or noteId of the subtree you want to visualize.

**Reply by devlinjunker** [reply]
Awesome! Thanks a lot! This works for what I was looking for!
